src/headers/validate_op.h

```diff
@@ -115,10 +115,12 @@ int OS_IsAfterTime(const char *time_str, const char *ossec_time) __attribute__((
  *      weekdays, weekends, monday, tuesday, thursday,..
  *      monday,tuesday
  *      mon,tue wed
- * @param day_str Day to be validated.
+ * @param[in] day_str Day to be validated.
```

Comment:
`@param [in]`

Reply:
Following doxygen documentation
https://www.doxygen.nl/manual/commands.html#cmdparam
There is no blank space between the parameter and the dir attribute
It can also be seen in our external libraries, i.g.
/src/external/libyaml/include/yaml.h
but it's confusing, in other places they use a blank space,  such as
/src/config/syscheck-config.h